data.sql

```diff
@@ -55,3 +55,38 @@ UPDATE animals SET owner_id = (CASE
                                 WHEN name IN ('Charmander', 'Squirtle', 'Blossom', 'Angemon', 'Boarmon') THEN 4
                                 ELSE 5
                               END);
+
+
```

Comment:
![image](https://github.com/ahmidou09/Vet-clinic-database/assets/93039359/9e051adb-70c7-4df5-ae46-f67942b31fb0)

- [ ] It would be nice if you fix the highlighted link in the image because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍